### PR TITLE
Added optional selector

### PR DESCRIPTION
**Pull request by githrdw**
Added font-face to import custom fonts

- [x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x] My pull request targets the `master` branch of freeCodeCamp.
- [x] None of my changes are plagiarized from another source without proper attribution.
- [x] My article does not contain shortened URLs or affiliate links.

### CODE REVIEW

File: guide/english/css/selectors/pseudo/optional/index.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+}
+
+input:optional {
+  background-color: red;
+}
+```
+
+#### More Information:
+* [MDN Web Docs](https://developer.mozilla.org/en-US/docs/Web/CSS/:optional)
+* [W3 Schools](https://www.w3schools.com/cssref/sel_optional.asp
```
Comment (co16353sidak) [inline guide/english/css/selectors/pseudo/optional/index.md]: Add a closing bracket `)` at then end of the link to complete the GitHub markdown syntax for a link.

**Review by theNaut1lus — APPROVED** [review]
@RandellDawson needs second opinion on directory structure.
Otherwise small typographical error, can be merged post rectification.

**Comment by githrdw** [comment]
> @RandellDawson needs second opinion on directory structure.
> Otherwise small typographical error, can be merged post rectification.

Sorry, done

**Comment by RandellDawson** [comment]
Thank you for your contribution to the page! 👍
We're happy to accept these changes, and look forward to future contributions. 🎉

**Pull request merged**